src/config.coffee

```diff
@@ -16,11 +16,11 @@ TODO rename
 @reserved = "res" # prefix var name
 @router_enum = "router_enum"
 
-@int_type_list = ["int"]
+@int_type_list = ["int", "signed_number"]
```

Comment:
Dangerous code. int_type_list is for types that are really exist in solidity. signed_number is synthetic type for type inference, sometimes TI give up and this type leaks to translate_ligo. But we should handle this type explicitly at translate_ligo, and not include it here.